PACKAGE = "slam_navigator"
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("max_linear_speed", double_t, 0, "Max linear speed (m/s)", 1.5, 0.1, 5.0)
gen.add("max_angular_speed", double_t, 0, "Max angular speed (rad/s)", 1.0, 0.1, 3.0)
exit(gen.generate(PACKAGE, "navigation", "CanConfig"))